services/QuillLMS/app/controllers/api/v1/activities_controller.rb

```diff
@@ -85,6 +87,29 @@ def published_edition
     render json: {}
   end
 
+  def question_health
+    questions = @activity.data["questions"]
+    tool = CLASSIFICATION_TO_TOOL[ActivityClassification.find(@activity.activity_classification_id).key.to_sym]
+    questions_arr = questions.each_with_index.map { |q, i|
+      question_number = i + 1
+      question = Question.find_by(uid: q["key"])
+      data = question.data
+      health_dashboard = QuestionHealthDashboard.new(@activity.id, question_number, question.uid)
```

Comment:
From here, it looks like this line will fail if a question cannot be retrieved in line 95